internal/ibmcloud-janitor/resources/vpc_lbs.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type VPCLoadBalancer struct{}
+
+// Cleans up the load balancers in a given region
+func (VPCLoadBalancer) cleanup(options *CleanupOptions) error {
+	resourceLogger := logrus.WithFields(logrus.Fields{"resource": options.Resource.Name})
+	resourceLogger.Info("Cleaning up the load balancers")
+	client, err := NewVPCClient(options)
+	if err != nil {
+		return errors.Wrap(err, "couldn't create VPC client")
+	}
+
+	loadBalancers, _, err := client.ListLoadBalancers(&vpcv1.ListLoadBalancersOptions{})
```

Comment:
see if this accepts the resource group or any other options to list whatever needs to be deleted instead of listing entire thing.

Reply:
For options, we only have these
```
type ListLoadBalancersOptions struct {
    // A server-provided token determining what resource to start the page on.
    Start *string `json:"start,omitempty"`

    // The number of resources to return on a page.
    Limit *int64 `json:"limit,omitempty"`

    // Allows users to set headers on API requests
    Headers map[string]string
}
```

Reply:
Initial plan was to check with resource group similar to deleting instances & networks, but it isn't available

Reply:
Yes listing load balancer is not providing any filters.

If the intention is to delete all the load balancers, then need to take care of paging as well.
https://github.com/openshift/hypershift/blob/main/cmd/infra/powervs/destroy.go#L735

Reply:
Updated with changes, PTAL.